exercises/node-javascript/encryption-context-start/retrieve.js

```diff
@@ -0,0 +1,25 @@
+const { decryptStream, KmsKeyringNode } = require("@aws-crypto/client-node");
```

Comment:
@mattsb42-aws @lizroth Here is the start retrieve file.  The Encryption context verification needs to be added.